concepticondata/concepticon.tsv

```diff
@@ -3933,4 +3933,27 @@ ID	GLOSS	SEMANTICFIELD	DEFINITION	ONTOLOGICAL_CATEGORY	REPLACEMENT_ID
 3947	HONEYCOMB	Animals	A wax structure consisting of multiple small holes made by bees to store their honey.	Person/Thing	
 3948	SHOTGUN	Warfare and hunting	A long gun (used mainly for hunting) that shoots multiple small bullets at a time.	Person/Thing	
 3949	RED-FOOTED TORTOISE	Animals	A medium-sized species of tortoise (Chelonoidis Carbonarius), found in northern parts of South America.	Person/Thing	
-3950	LIBER (INNER BARK)	Agriculture and vegetation	The inner most layer of bark (particularly from a tree).	Person/Thing	
\ No newline at end of file
+3950	LIBER (INNER BARK)	Agriculture and vegetation	The inner most layer of bark (particularly from a tree).	Person/Thing	
+3951	PANDA	Animals	Also *giant panda*, a white-and-black mammal of the *Ailuropoda* family, native to eastern Asia, commonly referred to as a type of bear.	Person/Thing	
+3952	REFRIGERATOR	Modern world	A storage box, compartment or room to stock goods by means of mechanical refrigeration.	Person/Thing	
+3953	CHARITY	Social and political relations	A benevolent action in favor of the needy, or an activity or gift for the benefit of the public.	Property	
+3954	FRIENDSHIP (FEELING)	Emotions and values	A feeling of affection, good disposition, and sympathy experienced towards someone.	Property	
+3955	HOSTILITY	Emotions and values	An aversion, feeling of rivalry, or aggressiveness experienced by an individual or group towards another.	Property	
+3956	DISPUTE	Social and political relations	A disagreement or contentious argument between parties, typically in a political or ideological sense.	Action/Process	
+3957	SYSTEM	Cognition	A set of connected elements that form a whole by virtue of their interrelatedness and function.	Person/Thing	
+3958	SOCIETY	Social and political relations	A large social body brought together and organized under a common set of rules or principles.	Person/Thing	
+3959	RELATIONSHIP	Cognition	The way in which two elements or individuals are connected, usually in a social or affective sense.	Property	
+3960	IDENTITY	Cognition	A persisting state of being the same for an entity or someone, from an objective or a subjective standpoint. Commonly referring to an individual's self-sense of being as a distinct person.	Property	
+3961	TEAM	Social and political relations	A cooperative group of individuals working together for a common purpose.	Person/Thing	
+3962	DISCIPLINE	Emotions and values	The ability and training of self-control and diligence, usually in abiding by a set of rules.	Property	
```

Comment:
This concept occurs in only one other list.

Reply:
It's mapped to the verb meaning in Hale-1973-1798-770, so the description should start with "To" and the ontological category needs to be changed to Action/Process.

Reply:
Oh, I understand. I'll unmap Hale in that case. I'll leave the concept as is for future need, unless it's important to remove it right away.